http/headers/Set-Cookie.json

```diff
@@ -124,7 +124,7 @@
         },
         "Partitioned": {
           "__compat": {
-            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie#partitioned",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies",
```

Comment:
dito

Reply:
Fixed in [036c74a](https://github.com/mdn/browser-compat-data/pull/25563/commits/036c74a1d18f9d6d352e80cc72572eb13e43513a).